metagraph/core/planning.py

```diff
@@ -197,12 +207,7 @@ def __repr__(self):
                 else:
                     s.append(f"** {varname} **")
                     anni = sig.parameters[varname].annotation
-                    if type(anni) is Wrapper:
-                        s.append(f"{anni.__name__}")
-                    elif type(anni) is type:
-                        s.append(f"{anni.__name__}")
-                    else:
-                        s.append(f"{anni.__class__.__name__}")
+                    s.append(self.__class__.string_for_annotation(anni))
```

Comment:
`self` has access to all classmethods and staticmethods. No need to use `self.__class__` here.

Reply:
https://github.com/metagraph-dev/metagraph/pull/120/commits/3022582fd357e1d6cd330caa32cd984df3d0fc6f removes `self.__class__`.